data/base-sigils.yaml

```diff
@@ -0,0 +1,5503 @@
+    - 894
+    - 895
+    - 898
+    - 899
+    - 901
+    - 906
+    - 908
+    - 909
+    - 910
+    - 911
+    - 916
+    - 919
+    - 924
+    - 925
+    - 929
+    - 934
+    - 936
+    - 942
+    summer: 
+    winter:
+    - 730
+    - 740
+    - 743
+    - 756
+    - 762
+    - 769
+    - 771
+    - 781
+    - 797
+    - 798
+    - 803
+    - 804
+    - 808
+    - 813
+    - 817
+    - 821
+    - 847
+    - 854
+    - 857
+    - 866
+    - 875
+    - 882
+    - 884
+    - 903
+    - 904
+    - 905
+    - 907
+    - 917
+    - 927
+    - 932
+    - 937
+    - 954
+    - 956
+    - 7905
+    - 619
+    - 622
+    - 623
+    - 625
+    - 629
+    - 632
+    - 638
+    - 639
+    - 646
+    - 647
+    - 648
+    - 649
+    - 653
+    - 654
+    - 659
+    - 663
+    - 674
+    - 701
+    - 711
+    - 724
+    - 726
+    - 727
+    - 1036
+    - 9905
+    - 13094
+    - 13096
+    - 13099
+  nurture:
+    spring:
+    - 826
+    - 828
+    - 829
+    - 832
+    - 957
+    - 1387
+    - 1389
+    - 1399
+    - 1400
+    - 1405
+    - 1410
+    - 1411
+    - 1413
+    - 1419
+    - 1421
+    - 1425
+    - 14036
+    - 14040
+    - 14042
+    - 14052
+    - 14062
+    - 14071
+    summer: 
+    winter:
+    - 795
+    - 796
+    - 810
+    - 824
+    - 831
+    - 838
+    - 840
+    - 931
+    - 944
+    - 945
+    - 946
+    - 615
+    - 617
+    - 618
+    - 626
+    - 627
+    - 628
+    - 631
+    - 635
+    - 637
+    - 641
+    - 642
+    - 643
+    - 645
+    - 650
+    - 652
+    - 656
+    - 658
+    - 675
+    - 681
+    - 722
+    - 723
+    - 725
+    - 1035
+    - 9906
+    - 13103
+  paradox:
+    spring:
+    - 729
+    - 731
+    - 733
+    - 734
+    - 741
+    - 744
+    - 747
+    - 748
+    - 753
+    - 755
+    - 757
+    - 761
+    - 767
+    - 772
+    - 773
+    - 777
+    - 778
+    - 780
+    - 784
+    - 788
+    - 789
+    - 790
+    - 791
+    - 793
+    - 794
+    - 799
+    - 802
+    - 814
+    - 835
+    - 836
+    - 839
+    - 849
+    - 859
+    - 864
+    - 871
+    - 872
+    - 876
+    - 878
+    - 879
+    - 881
+    - 887
+    - 889
+    - 891
+    - 896
+    - 900
+    - 912
+    - 918
+    - 921
+    - 922
+    - 939
+    - 941
+    - 943
+    - 955
+    - 7902
+    - 12271
+    - 12273
+    - 12275
+    summer: 
+    winter:
+    - 745
+    - 827
+    - 830
+    - 613
+    - 616
+    - 620
+    - 630
+    - 634
+    - 640
+    - 644
+    - 655
+    - 657
+    - 680
+    - 712
+    - 716
+    - 1039
+    - 1043
+    - 12569
+    - 13098
+  unity:
+    spring:
+    - 850
+    - 851
+    - 1391
+    - 1397
+    - 1404
+    - 1406
+    - 1409
+    - 1415
+    - 1416
+    - 1417
+    - 1420
+    - 14039
+    - 14048
+    - 14053
+    - 14054
+    - 14058
+    - 14059
+    - 14060
+    - 14063
+    - 14064
+    - 14065
+    - 14067
+    - 14069
+    - 14070
+    - 14072
+    - 14073
+    - 14075
+    - 14077
+    - 14078
+    summer: 
+    winter:
+    - 825
+    - 717
+    - 1040
+    - 1041
+riverhaven:
```

Comment:
🧹  Would you mind spelling the town names the same was as they are listed in https://github.com/rpherbig/dr-scripts/blob/master/data/base-town.yaml as a consistency thing?

Reply:
updated the capitalization of the cities